components/ActionsDropdown/ActionsDropdown.tsx

```diff
@@ -71,13 +85,42 @@ export const ActionsDropdown: React.FC<IActionsDropdownComponentProps> = withMoo
 		<Transition addEndListener={() => undefined} unmountOnExit in={props.isOpen} timeout={145}>
 			{(state) => {
 				const ActionDropdownComponent = DropdownComponents[state];
+				
 				if (ActionDropdownComponent) {
 					return (
-						<ActionDropdownComponent boxShadow="base" {...props}>
-							<Box position="absolute" h={30} top={-30} left={0} right={0} />
-							{!hasArrow && <ActionsDropdownArrow boxShadow="base" />}
-							{recurseChildren(props.children)}
-						</ActionDropdownComponent>
+						<>
+							<ActionDropdownComponent 
+								boxShadow="base"
+								position={['relative', 'relative', 'fixed', 'fixed']}
+								borderRadius={['2px', '2px', '8px', '8px']}
+								bottom={['unset', 'unset', '73px', '73px']}
+								left={['unset', 'unset', '10px', '10px']}
+								right={['unset', 'unset', '10px', '10px']}
+								{...props}
+							>
+								<Box position="absolute"  />
+								{!hasArrow && <ActionsDropdownArrow boxShadow="base" />}
+									{recurseChildren(props.children)}					
+							</ActionDropdownComponent>
+							<ActionsDropdownButtonClose 
```

Comment:
so here you can use StyledActionsDropdownButtonClose directly. please remove the styling props that dont use media queries, instead just use them in the StyledActionsDropdownButtonClose styled component within styles.ts. I think there's also duplicate styling here. it would just generate extra classes.